evalai/utils/submissions.py

```diff
@@ -280,6 +280,28 @@ def display_submission_result(submission_id):
             )
         )
 
+def display_submission_stderr(submission_id):
+    """
+    Function to display stderr file of a particular submission in Terminal output
+    """
+    try:
+        response = submission_details_request(submission_id).json()
+        file_url = requests.get(response['stderr_file']).text
+        with open(file_url, "r") as fr:
+            try:
+                file_contents = fr.read()
+                print (file_contents)
+                fr.close()
+            except (OSError, IOError) as e:
+                echo(e)
+    except requests.exceptions.MissingSchema:
+        echo(
+            style(
+                "\nThe Submission is yet to be evaluated.\n",
+                bold=True,
+                fg="yellow",
+            )
+        )
```

Comment:
Can you explain why have you used `requests.exceptions.MissingSchema`.
Going by [docs](https://requests.readthedocs.io/en/master/_modules/requests/exceptions/),`MissingSchema` is for  `The URL schema (e.g. http or https) is missing.` exception.

Reply:
@vkartik97, I used this exception because as per your earlier note on my previous task “display stdout message in terminal”  the stdout file is generated after the submission evaluation is completed.  So if the submission id is not available means that it is supposed to be "yet to be evaluated". So I thought that would be right exception error. I just followed the same def function “display_submission_result” code from submissions.py for this one as well. Let me know if you don’t want this exception to be shown, then what exception you are suggesting. Thank you

Reply:
You can use other exception or use `status` data from the payload